core/modules/widgets/action-deletefield.js

```diff
@@ -59,7 +59,7 @@ DeleteFieldWidget.prototype.invokeAction = function(triggeringWidget,event) {
 		tiddler = this.wiki.getTiddler(self.actionTiddler),
 		removeFields = {},
 		hasChanged = false;
-	if(this.actionField && tiddler) {
```

Comment:
We prefer !== unless the != operator is specifically required, in which case we should add a comment explaining why.

Reply:
uuups